scripts/devops_tasks/build_packages.py

```diff
@@ -22,7 +22,17 @@
 sys.path.append(tox_path)
 from sanitize_setup import process_requires
 
-def build_packages(targeted_packages, distribution_directory, is_dev_build = False):
+
+def str_to_bool(input_string):
+    if isinstance(input_string, bool):
+        return input_string
+    if input_string.lower() in ("true", "t", "1"):
+        return True
+    elif input_string.lower() in ("false", "f", "0"):
```

Comment:
nit: Should this be changed as default "else" or  add a default handle for "else" . Ideally not required since it has to be either `true` or `false`

Reply:
nah man I'll take it. Gotchu.